account-and-billing/account-and-billing.helper.ts

```diff
@@ -0,0 +1,172 @@
+import { StepLogger } from '../../../../core/logger/step-logger';
+import { AccountAndBillingPage } from './account-and-billing.po';
+import { AccountAndBillingConstant } from './account-and-billing.constant';
+import { ExpectationHelper } from '../../../components/misc-utils/expectation-helper';
+import { CommonPageHelper } from '../common/common-page.helper';
+import { PageHelper } from '../../../components/html/page-helper';
+
+const { text: textConst } = AccountAndBillingConstant.attributes;
+const labelsPO = AccountAndBillingPage.labels;
+const linksPO = AccountAndBillingPage.links;
+
+export class AccountAndBillingHelper {
+    static async verifyPage() {
+        await CommonPageHelper.switchToPageFrame();
+        StepLogger.subVerification('Account and Billing page is displayed');
+        await ExpectationHelper.verifyDisplayedStatus(
+            labelsPO.accountAndBillingPageHeader,
+            textConst.accountAndBilling);
+    }
+
+    static async verifyPastPaymentsHeader() {
+        StepLogger.subVerification('Past Payments header is displayed');
+        await ExpectationHelper.verifyDisplayedStatus(
+            labelsPO.pastPaymentsHeader,
+            textConst.pastPayments);
+    }
+
+    static async verifyAccountIdHeader() {
+        StepLogger.subVerification('Account ID header is displayed');
+        await ExpectationHelper.verifyDisplayedStatus(
+            labelsPO.accountIdHeader,
+            textConst.accountId);
+    }
+
+    static async verifyCustomerSupportHeader() {
+        StepLogger.subVerification('Customer Support header is displayed');
+        await ExpectationHelper.verifyDisplayedStatus(
+            labelsPO.customerSupportHeader,
+            textConst.customerSupport);
+    }
+
+    static async verifyPrimaryContactHeader() {
+        StepLogger.subVerification('Primary Contact header is displayed');
+        await ExpectationHelper.verifyDisplayedStatus(
+            labelsPO.primaryContactHeader,
+            textConst.primaryContact);
+    }
+
+    static async verifyExportDataHeader() {
+        StepLogger.subVerification('Export Data header is displayed');
+        await ExpectationHelper.verifyDisplayedStatus(
+            labelsPO.exportDataHeader,
+            textConst.exportData);
+    }
+
+    static async verifyDownloadDataLink() {
+        StepLogger.subVerification('Download Data link is displayed');
+        await ExpectationHelper.verifyDisplayedStatus(
+            linksPO.downloadDataLink,
+            textConst.downloadLink);
+    }
+
+    static async verifyCloseAccountLink() {
+        StepLogger.subVerification('Close Account link is displayed');
+        await ExpectationHelper.verifyDisplayedStatus(
+            linksPO.closeAccount,
+            textConst.closeAccount);
+    }
+
+    static async verifyCloseAccountInformation() {
+        StepLogger.subVerification('"Closed accounts cannot be recovered" label is displayed');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/davidgunadi/hello-world/pullRequest?prid=8400120)
```suggestion
        StepLogger.subVerification("\"Closed accounts cannot be recovered\" label is displayed");
```